src/test/ui/attributes/register-attr-tool-prelude.rs

```diff
@@ -7,7 +7,7 @@
 #[no_implicit_prelude]
 mod m {
     #[attr] //~ ERROR cannot find attribute `attr` in this scope
-    #[tool::attr] //~ ERROR failed to resolve: use of undeclared type or module `tool`
+    #[tool::attr] //~ ERROR failed to resolve: use of undeclared crate, module, or type `tool`
```

Comment:
For #75987, you don't need the full error to match on:
```suggestion
    #[tool::attr] //~ ERROR failed to resolve: use of undeclared crate
```